common/v2/features/Settings/Settings.tsx

```diff
@@ -4,13 +4,24 @@ import styled from 'styled-components';
 import translate from 'v2/translations';
 
 import { IS_MOBILE } from 'v2/utils';
-import { BREAK_POINTS, MIN_CONTENT_PADDING } from 'v2/theme';
-import { AddressBookContext, SettingsContext, StoreContext } from 'v2/services/Store';
-import { AccountList, FlippablePanel, TabsNav } from 'v2/components';
-import { AddressBookPanel, AddToAddressBook, GeneralSettings, DangerZone } from './components';
+import { BREAK_POINTS } from 'v2/theme';
+import {
+  AddressBookContext,
+  NetworkContext,
+  NetworkUtils,
+  SettingsContext,
+  StoreContext
+} from 'v2/services/Store';
+import { AccountList, FlippablePanel } from 'v2/components';
+import { NetworkId } from 'v2/types';
+import { CustomNodeConfig } from 'v2/types/node';
+import { DEFAULT_NETWORK, IS_ACTIVE_FEATURE } from 'v2/config';
 
 import settingsIcon from 'common/assets/images/icn-settings.svg';
-import { IS_ACTIVE_FEATURE } from 'v2/config';
+import NetworkNodes from './components/NetworkNodes';
+import AddOrEditNetworkNode from './components/AddOrEditNetworkNode';
+import { AddressBookPanel, AddToAddressBook, GeneralSettings, DangerZone } from './components';
+import MobileNavBar from 'v2/components/MobileNavBar';
```

Comment:
Please add `MobileNavBar` ton `index.ts` in components folder and import it in the import `v2/components`